doc/developer-guide/cripts/cripts-certs.en.rst

```diff
@@ -0,0 +1,172 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. _cripts-certs:
+
+Certificates
+************
+
+Cripts provides a set of convenient classes for introspection into the various
+TLS certificates that are used. This includes both the server certificates used
+to establish a TLS connections, as well as any client certificates use for
+mutual TLS.
```

Comment:
...are used. This includes (--> These include) (because classes are plural)
...any client certificates use (--> used) for

Reply:
Fixed